src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
```

Comment:
Avoid using tag names for styling (except `html` and `body`). If you need to apply a reset or global styles, it's acceptable, but if you're targeting a specific section or component, it's better to use classes to avoid potential conflicts and enhance maintainability.